src/Keys.php

```diff
@@ -39,7 +39,7 @@ public function getPublicKey($priv_hex)
         $priv = $ec->keyFromPrivate($priv_hex);
         $pub_hex = $priv->getPublic(true, 'hex');
 
-        return $pub_hex;
+        return substr($pub_hex, 2); // remove compression prefix 02 | 03
```

Comment:
Hmm, more a fan of putting comments above the code - unless it's a specific code style ing

Reply:
:D No it is not a specific code styling, just too late at night